### DISCUSSION TITLE
Question: Docker LXC Script wrong Port?

### DISCUSSION THREAD

**Discussion by Momentum6890** [tteck/Proxmox#1839] [General]
Hello tteck, first of all I want to say your scripts are really amazing, they make it really super easy for a Proxmox/Linux beginner like me :D

My question is, is the port wrong on the docker script?

The port 9000 is not reachable. Only the default port of Portainer (9443) is reachable via https.

http and port 9000 is not reachable for me.

**Comment by mrtrofimov** [comment]
@Momentum6890 Oh! Man! You are right! I've reinstalled Docker several times trying to find out what is wrong ))

**Reply by Momentum6890** [reply]
Me too :P

**Comment by tteck** [comment]
https://github.com/tteck/Proxmox/issues/1763

https://github.com/tteck/Proxmox/commit/9dc43c396f96f7916e07b7c8207bfca185a22e5d

**Reply by Momentum6890** [reply]
Oh sorry, i searched for docker but overlooked this post unfortunately, but thanks :)

**Comment by mrtrofimov** [comment]
@tteck May be should correct description as well? [https://tteck.github.io/Proxmox/](url)
_"Portainer Interface: IP:9000"_
